src/client/components/pages/entities/work-table.js

```diff
@@ -59,6 +59,7 @@ function WorkTableRow({showAddedAtColumn, work, showCheckboxes, selectedEntities
 				<a href={`/work/${work.bbid}`}>{name}</a>
 				{disambiguation}
 			</td>
+			<td>{work.author && <a href={`/author/${work.author.bbid}`}>{work.author.defaultAlias.name}</a>}</td>
```

Comment:
As mentioned in the ORM PR, we will want an array or authors as works can have multiple "author wrote work" relationships.